XRTK-Core/Packages/com.xrtk.core/Definitions/BaseMixedRealityProfile.cs

```diff
@@ -1,19 +1,29 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+﻿// Copyright (c) XRTK. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
 using UnityEngine;
+using UnityEngine.Serialization;
 
 namespace XRTK.Definitions
 {
     public abstract class BaseMixedRealityProfile : ScriptableObject
     {
         [SerializeField]
-        private bool isCustomProfile = true;
+        [FormerlySerializedAs("isCustomProfile")]
+        [FormerlySerializedAs("isDefaultProfile")]
```

Comment:
Was it really ever serialized as "isDefaultProfile"? Don't remember the attribute used to be there previously, so why add it now?

Reply:
From the beginnings, CustomProfiles were one's created by the customer, this logically turns it on its head to state if the profile is a default provided by the XRTK or not.
This is purely about keeping maintenance simple for the content the XRTK provides

Reply:
Sure I get that. I am just saying `[FormerlySerializedAs("isDefaultProfile")]` doesn't make any sense to me here. But I might be missing a piece of the cake here.

Reply:
FormalerlySerialzedAs is needed so that it doesn't break existing Profiles that have been created in host projects.  This will get removed at a later date when it's unlikely to be an issue.
As Config files are opened/used, Unity will recognise the old name and upgrade the scriptable object automatically.  Else, the config would break when opened :D

Reply:
We are talking different things here. I know what `FormalerlySerialzedAs` does and what it's used for. I am just saying the former name of this field was `isCustomProfile` and NOT "isDefaultProfile". And since the `[FormerlySerializedAs("isDefaultProfile")]` is not there in the dev branch it was added as part of this PR. I am not seeing any use in it since obviously there is no requirement for it for backwards compatibility.

Reply:
Ahh, I see your point, @StephenHodgson are we targetting the wrong attribute here?

Although in testing, it hasn't caused an issue surprisingly :S

Reply:
We moved from targeted serialized types to a generic one. So each profile had slightly different names for the same field. I renamed them, and this should prevent them from breaking.

Reply:
I am a little bit nervous that this may break existing profiles, so if it does let me know. 

I no longer have an active project to test this with.

Reply:
@FejZa can you test using a copy of your hands demonstration project, or another XRTK project?

Reply:
@greyfoxlinux should also test his project as well